apps/bixarena/app/bixarena_app/model/error_handler.py

```diff
@@ -30,13 +30,13 @@ def get_battle_round_limit_message() -> str:
     Provide a user-facing message when the battle round cap is reached.
 
     Returns:
-        Markdown formatted message consistent with other error responses.
+        HTML formatted message for system message display.
     """
     message = (
-        "**You've reached the round limit for this battle.**\n\n"
+        "<strong>You've reached the round limit for this battle.</strong><br><br>"
```

Comment:
Make the system bubble more compact by removing the empty lines between the sentences.